doc/sphinx/source/recipes/recipe_python.rst

```diff
@@ -0,0 +1,61 @@
+.. _recipe_python:
+
+Example recipe Python
```

Comment:
In the slightly longer term, we could maybe rename this to `Example recipes` and also add the other example diagnostics in the same document? Especially recipe_ncl.yml, recipe_r.yml and recipe_julia.yml

Reply:
Yes, good idea. Shall I make an issue about this?

Reply:
https://github.com/ESMValGroup/ESMValTool/issues/1861